peer/brontide.go

```diff
@@ -703,6 +703,23 @@ func (p *Brontide) Start() error {
 
 	p.startTime = time.Now()
 
+	// Before launching the writeHandler goroutine, we send any channel
+	// sync messages that must be resent for borked channels. We do this to
+	// avoid data races with WriteMessage & Flush calls.
+	if len(msgs) > 0 {
+		p.log.Infof("Sending %d channel sync messages to peer after "+
+			"loading active channels", len(msgs))
+
+		// Send the messages directly via writeMessage and bypass the
+		// writeHandler goroutine.
+		for _, msg := range msgs {
+			if err := p.writeMessage(msg); err != nil {
+				return fmt.Errorf("unable to send reestablish"+
```

Comment:
nit: space at end of comment

Reply:
a little confused, where should I add the space?

Reply:
```suggestion
				return fmt.Errorf("unable to send reestablish "+
```

Reply:
Ended up breaking the line since adding a space would put it at 81 chars